contrastive_training/contrastive_main.py

```diff
@@ -49,23 +49,23 @@
 #input_mash_mat = torch.from_numpy(input_mash_mat_np).to(dtype=dtype,device=device)
 #data_size = input_mash_mat.shape[0]
 
-input_mash_mat_subsample_np = np.load("/contrastive_training/input_features.npy")
+input_mash_mat_subsample_np = np.load("C:/Users/Zhiwen Yan/Desktop/23summer/input_features.npy")
 input_mash_mat_subsample = torch.from_numpy(input_mash_mat_subsample_np).to(dtype=dtype,device=device)
 
 input_mash_mat_np = input_mash_mat_subsample_np[np.arange(training_size)*8+7]
 input_mash_mat = torch.from_numpy(input_mash_mat_np).to(dtype=dtype,device=device)
 data_size = input_mash_mat.shape[0]
 
 #similarity matrix
-sim_mat_np = np.load("similarity_mat.npy")
+sim_mat_np = np.load("C:/Users/Zhiwen Yan/Desktop/23summer/similarity_mat.npy")
 #data_size = sim_mat_np.shape[0]
 sim_mat_np = (np.ones((data_size,data_size))-np.eye(data_size))*sim_mat_np
 sim_mat_np = sim_mat_np/sim_mat_np.max()
 sim_mat = torch.from_numpy(sim_mat_np).to(dtype=dtype,device=device)
 assert data_size == sim_mat.shape[0]
 
 #similarity matrix with subsampling
-sim_mat_subsample_np = np.load("similarity_mat_subsample.npy")
+sim_mat_subsample_np = np.load("C:/Users/Zhiwen Yan/Desktop/23summer/similarity_mat_subsample.npy")
```

Comment:
Hard coded path.

Reply:
Contrastive part has not been generalized, just a test version on local machine.